module_utils/certificate/providers/certmonger.py

```diff
@@ -141,5 +141,8 @@ def request_certificate(self):
             # Don't attempt to renew when near to expiration
             command += ["-R"]
 
+        # Set certificate key size
+        command += ["-g", str(self.module.params.get("key_size"))]
```

Comment:
Is `-g` a required argument?

Reply:
It's not required but I'm setting it always to ensure `key_size` will have the default value we chose (currently 2048).